docker/Dockerfile.distroless

```diff
@@ -1,5 +1,5 @@
 # prepare a distroless source context to copy files from
-FROM gcr.io/distroless/static-debian11@sha256:21d3f84a4f37c36199fd07ad5544dcafecc17776e3f3628baf9a57c8c0181b3f as distroless_source
+FROM gcr.io/distroless/static-debian11@sha256: as distroless_source
```

Comment:
This doesn't look right.

Reply:
Not sure why these are wrong, but running commands themselves so no changes required.